local-setup/launch.py

```diff
@@ -41,7 +41,7 @@ def run_node(config):
 
 def run_worker(config, i: int):
     log = open(f'{log_dir}/worker{i}.log', 'w+')
-    w = setup_worker(f'/tmp/w{i}', config["source"], log)
+    w = setup_worker(f'tmp/w{i}', config["source"], log)
```

Comment:
So this will create `tmp/` directories in the repository directory? If so, can you add that directory to `.gitignore` ?

Reply:
Yes. If two people use the script on the same time on the same time, only one will be able to actually execute anything. The second one will get an Permission denied because the owner of /tmp/w1 is someone else.

Reply:
Alright, the better way would be to introduce some randomness in the `/tmp` dir creation, but as the python stuff will be obsolete anyhow, I am fine.